docs/manual/src/docs/asciidoc/_includes/servlet/authentication/index.adoc

```diff
@@ -33,6 +33,7 @@ This also gives a good idea of the high level flow of authentication and how pie
 * <<servlet-rememberme, Remember Me>> - How to remember a user past session expiration
 * <<servlet-jaas, JAAS Authentication>> - Authenticate with JAAS
 * <<servlet-openid,OpenID>> - OpenID Authentication (not to be confused with OpenID Connect)
+// FIXME: The one above is deprecated. Should it be removed from here as well?
```

Comment:
We do not need to remove it here. We still want users to find it and figure out it has been deprecated.
```suggestion
```